bindings/python/fdb/impl.py

```diff
@@ -736,6 +745,14 @@ def wait(self):
         # the KVs on the python side and in most cases we are about to
         # destroy the future anyway
 
+class FutureKeyArray(Future):
+    def wait(self):
+        self.block_until_ready()
+        ks = ctypes.pointer(KeyStruct())
+        count = ctypes.c_int()
+        self.capi.fdb_future_get_key_array(self.fpointer, ctypes.byref(ks), ctypes.byref(count))
+        return ([ctypes.string_at(x.key, x.key_length) for x in ks[0:count.value]], count.value)
```

Comment:
I think this also should probably just return the array of keys rather than a tuple that includes the count.

Reply:
Done.